src/main/java/com/teamknp/hotel/config/StaticResourceLocationConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package com.teamknp.hotel.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+
+@Configuration
+public class StaticResourceLocationConfiguration {
+    @Bean
+    WebMvcConfigurer configurer () {
+        return new WebMvcConfigurerAdapter() {
```

Comment:
Nie trzeba się bawić z takimi dziwnymi konstrukcjami to jest. Tworzyć config, który ma funkcje/bean, która tworzy anonimową klasę wewnętrzną, który rozszerza WebMvcConfigurerAdapter. W prawidłowy sposób to po prostu powinna być klasa. która implementuje interfejs WebMvcConfigurerAdapter, ale powinna mieć adnotacje @Component.